ra/ra_test.go

```diff
@@ -437,6 +437,122 @@ func TestNewAuthorization(t *testing.T) {
 	t.Log("DONE TestNewAuthorization")
 }
 
+func TestReuseAuthorization(t *testing.T) {
+	_, sa, ra, _, cleanUp := initAuthorities(t)
+	defer cleanUp()
+
+	// Turn on AuthZ Reuse
+	ra.reuseValidAuthz = true
+
+	// Create one finalized authorization
+	AuthzFinal.RegistrationID = Registration.ID
+	AuthzFinal, err := sa.NewPendingAuthorization(ctx, AuthzFinal)
```

Comment:
We should probably call this by a new name to avoid confusion. Also, the pattern this test uses of modifying global state is sub-optimal. Filed a ticket to deal with that separately: #1922